index.jsx

```diff
@@ -109,7 +109,7 @@ class StarRatingComponent extends Component {
     // populate stars
     let starNodes = [];
 
-    for (let i = starCount; i > 0; i--) {
+    for (let i = starCount; i > 0; i - 1) {
```

Comment:
what's the point of these changes?

Reply:
This was based on my .eslintrc temporarily being misconfigured, and having ESLint complain about this rule:
https://eslint.org/docs/rules/no-plusplus
It seems sensible to me and is functionally equivalent, but I'm happy to revert this change.